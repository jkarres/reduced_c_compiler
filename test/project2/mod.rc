// testing %

function : void main() {
  int a = 10;
  int b = 9;
  int c = 8;

  cout << 1 << " " << a % a << endl;
  cout << 2 << " " << a % b << endl;
  cout << 3 << " " << a % c << endl;

  cout << 4 << " " << b % a << endl;
  cout << 5 << " " << b % b << endl;
  cout << 6 << " " << b % c << endl;

  cout << 7 << " " << c % a << endl;
  cout << 8 << " " << c % b << endl;
  cout << 9 << " " << c % c << endl;
}

