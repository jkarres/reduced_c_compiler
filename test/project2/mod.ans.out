1 0
2 1
3 2
4 9
5 0
6 1
7 8
8 8
9 0
